tests/integration/tests/remote_istiod_test.go

```diff
@@ -89,6 +89,9 @@ func TestRemoteIstiod(t *testing.T) {
 	conf.ExternalServices.Istio.Registry = &config.RegistryConfig{
 		IstiodURL: "http://istiod-debug.istio-system:9240",
 	}
+	// Since the current operator version doesn't support ExternalServices.Istio.Registry,
+	// we need to modify the ConfigMap directly
+	instance.UseKialiCR = false
```

Comment:
Why doesn't the operator support `ExternalServices.Istio.Registry`? It's a valid Kiali config option: https://github.com/kiali/kiali/blob/bde18d5b0628d9a0644d09768f344b143e7eeb89/config/config.go#L359